public/index.html

```diff
@@ -43,24 +39,105 @@ <h5 id="campaignType">Campaign Type</h5>
     <div id="missionList"></div>
   </div>
 
+  <div id="dataDiv">test</div>
+
   <script>
     $.getJSON('/api/data', function(data) {
       $('#campaignName').text(data.name);
-      $('#campaignType').text(data.campaign_type);
+      $('#campaignType').text(data.campaignType);
 
       for (var index in data.completedMissions) {
         var mission = data.completedMissions[index];
         $('#missionList').append('<div class="completedMission">' + mission + '</div>');
       };
 
-      $('#imperialPlayer').text(data.imperialPlayer.name);
+      // Add infor for Imperial player
+      var imperialPlayer = $("<div/>").attr({
+        class:"well text-left",
```

Comment:
nit: all of your new code should have a space after the colon

that is

```
{
  key: "value",
}
```

vs

```
{
  key:"value",
}
```